aws-codeartifact-repository/aws-codeartifact-repository.json

```diff
@@ -0,0 +1,84 @@
+{
+  "typeName": "AWS::CodeArtifact::Repository",
+  "description": "The resource schema to create a CodeArtifact repository.",
+  "sourceUrl": "https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-codeartifact",
+  "properties": {
+    "RepositoryName": {
+      "description": "The name of the repository.",
+      "type": "string",
+      "pattern": "[a-z][a-z0-9\\-]{0,48}[a-z0-9]",
+      "minLength": 2,
+      "maxLength": 100
+    },
+    "DomainName": {
+      "description": "The name of the domain that contains the repository.",
+      "type": "string",
+      "pattern": "[a-z][a-z0-9\\-]{0,48}[a-z0-9]",
+      "minLength": 2,
+      "maxLength": 50
+    },
+    "DomainOwner": {
+      "description": "The 12-digit account ID of the AWS account that owns the domain.",
+      "pattern": "[0-9]{12}",
+      "type": "string"
+    },
+    "AdministratorAccount": {
+      "description": "The 12-digit account ID of the AWS account that manages the repository.",
+      "pattern": "[0-9]{12}",
+      "type": "string"
+    },
+    "Description": {
+      "description": "A text description of the repository.",
+      "type": "string",
+      "maxLength": 100
+    },
```

Comment:
This isn't right. A repo description can be much longer than this, please update based on our model.

Reply:
missed a 0, thanks for catching